registration/tests.py

```diff
@@ -459,6 +589,112 @@ def test_jembi_post_json(self):
                       body='jembi_post_json task', status=201,
                       content_type='application/json')
 
-        task_response = jembi_post_json.apply_async(
+        task_response = tasks.jembi_post_json.apply_async(
             kwargs={"registration_id": registration.data["id"]})
         self.assertEqual(task_response.get(), 'jembi_post_json task')
+
+
+class TestUpdateCreateVumiContactTask(AuthenticatedAPITestCase):
+
+    def test_update_vumi_contact(self):
+        registration = self.make_registration(
+            post_data=TEST_REG_DATA["clinic_self"])
+        client = self.make_client()
+        self.make_existing_contact({
+            u"key": u"knownuuid",
+            u"msisdn": u"+27001",
+            u"user_account": u"knownaccount",
+            u"extra": {}
+        })
+
+        contact = tasks.update_create_vumi_contact.apply_async(
+            kwargs={"registration_id": registration.data["id"],
+                    "client": client})
+        self.assertEqual(contact.get()["msisdn"], "+27001")
```

Comment:
This works of course, but you could simplify this for yourself but doing `result = contact.get()` and then just using `result["key"], "expected"` throughout.